circle.yml

```diff
@@ -2,7 +2,7 @@ machine:
   environment:
     NODE_ENV: test
   node:
-    version: 0.12.6
+    version: 5.1.0
```

Comment:
@jellymann bumped to 5.9.0 (aligning with what is available from https://hub.docker.com/r/nodesource/trusty/tags/)